### ISSUE TITLE
Breaking change between 0.1 and 0.4, incorrect docs

### ISSUE THREAD

**Issue by dylanjha** [Patreon/patreon-js#21]
The exposed API for the package has changed

0.1.0

```js
const patreon = require('patreon')

Object.keys(patreon)   // ['oauth', 'default']
typeof patreon.oauth   // 'function'
typeof patreon.default // 'function'
```

0.4.0

```js
const patreon = require('patreon')

Object.keys(patreon)       // [ 'oauth', 'patreon', 'jsonApiURL' ]
typeof patreon.oauth       // 'function'
typeof patreon.patreon     // 'function'
typeof patreon.jsonApiURL  // 'function'
```

Despite the new api, the docs have not been updated so the examples in Usage are not accurate. Using the example:

```js
var patreon = require('patreon')
var patreonAPI = patreon.default
var patreonOAuth = patreon.oauth

patreonAPI(<access token>)
// ERROR patreonAPI is not a function
```

**Comment by nperickson** [comment]
Hey @dylanjha -

Thanks for reporting this. I've made a task for us to get this updated.

**Comment by emosesPatreon** [comment]
This is fixed in the README as of #22
